### PR TITLE
Bump libraries and run centaur bcs.

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

File: cloud-nio/cloud-nio-impl-drs/src/main/scala/cloud/nio/impl/drs/DrsPathResolver.scala [hunk 1]
```diff
@@ -15,10 +15,6 @@ import org.apache.http.impl.client.HttpClientBuilder
 import org.apache.http.util.EntityUtils
 import org.apache.http.{HttpResponse, HttpStatus}
 
-//Do not remove this import. This import is required to compile, but IntelliJ doesn't correctly recognize that.
-import cats.syntax.functor._
-
-
```
Comment (mcovarr) [inline cloud-nio/cloud-nio-impl-drs/src/main/scala/cloud/nio/impl/drs/DrsPathResolver.scala]: ?

File: dockerHashing/src/main/scala/cromwell/docker/DockerInfoActor.scala [hunk 2]
```diff
@@ -112,7 +113,7 @@ final class DockerInfoActor(
   /*
    * Sends back responses and adds to cache in case of success. Used as the sink for each registry stream.
    */
-  val streamSink = fs2.Sink[IO, (DockerInfoResponse, DockerInfoContext)] {
+  val streamSink: Pipe[IO, (DockerInfoResponse, DockerInfoContext), Unit] = _ evalMap {
```
Comment (mcovarr) [inline dockerHashing/src/main/scala/cromwell/docker/DockerInfoActor.scala]: ici n'est pas une sink.

File: docs/api/RESTAPI.md [hunk 3]
```diff
@@ -577,33 +617,7 @@ Returns a javascript file which will render a Gantt chart for the requested work
...
-|---|---|---|
-|**200**|Successful Request|[StatsResponse](#statsresponse)|
-
-
-#### Tags
-
-* Engine
-
-
-<a name="status"></a>
```
Comment (mcovarr) [inline docs/api/RESTAPI.md]: good catch

File: project/Dependencies.scala [hunk 4]
```diff
@@ -318,16 +235,27 @@ object Dependencies {
       exclude("com.google.guava", "guava-jdk5")
   )
 
+  /*
+  Used instead of `"org.lerch" % "s3fs" % s3fsV exclude("org.slf4j", "jcl-over-slf4j")`
+  org.lerch:s3fs:1.0.1 depends on a preview release of software.amazon.awssdk:s3.
+
+  Instead the code has been re-forked into this repo, just like many of the other FileSystemProvider extensions.
```
Comment (mcovarr) [inline project/Dependencies.scala]: 😮

File: src/ci/resources/bcs.jq [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+# Returns items under `key` that were created more than `seconds` ago.
+# For bcs jobs and clusters the key is usually `.`.
+# Expects `key` to be able to parse `{ Items: [ .Items[] | {Id, CreationTime} ] }`
+def printIdsMoreThanSecondsOld(key; seconds):
+    key
+    | .Items
+    | filterMoreThanSecondsOld(.CreationTime; seconds)
+    | .[]
+    | .Id;
```
Comment (mcovarr) [inline src/ci/resources/bcs.jq]: woah

File: cloud-nio/cloud-nio-impl-drs/src/main/scala/cloud/nio/impl/drs/DrsPathResolver.scala (hunk 1 above)
Comment (kshakir) [reply cloud-nio/cloud-nio-impl-drs/src/main/scala/cloud/nio/impl/drs/DrsPathResolver.scala]: I didn't check specifically what changed... but with the `cats` bump scalac no longer likes this import.

File: dockerHashing/src/main/scala/cromwell/docker/DockerInfoActor.scala (hunk 2 above)
Comment (kshakir) [reply dockerHashing/src/main/scala/cromwell/docker/DockerInfoActor.scala]: [For the curious.](http://www.dereferer.org/?https%3A%2F%2Fgithub%2Ecom%2Ffunctional%2Dstreams%2Dfor%2Dscala%2Ffs2%2Fpull%2F1386)

File: docs/api/RESTAPI.md (hunk 3 above)
Comment (kshakir) [reply docs/api/RESTAPI.md]: All of these weren't my changes. Over the next month or so look for an incoming PR to CI-verify that this file has been synced with cromwell.yaml.

File: project/Dependencies.scala (hunk 4 above)
Comment (kshakir) [reply project/Dependencies.scala]: Sub optimal, but I'm not aware of an actively maintained version of this code elsewhere.

Anyways, welcomed into the fold with all of the other NIO2 `FileSystemProvider` implementations already spread across this repo. [[1]](https://github.com/broadinstitute/cromwell/blob/fd5bdf873a9d9fae538c61e7e0945ce0d06a64b0/filesystems/oss/src/main/scala/cromwell/filesystems/oss/nio/OssStorageFileSystemProvider.scala#L20) [[2]](https://github.com/broadinstitute/cromwell/blob/develop/cloud-nio/cloud-nio-impl-drs/src/main/scala/cloud/nio/impl/drs/DrsCloudNioFileSystemProvider.scala#L15-L18) [[3]](https://github.com/broadinstitute/cromwell/blob/develop/cloud-nio/cloud-nio-impl-ftp/src/main/scala/cloud/nio/impl/ftp/FtpCloudNioFileSystemProvider.scala#L14)

File: src/ci/resources/bcs.jq (hunk 5 above)
Comment (mcovarr) [reply src/ci/resources/bcs.jq]: TIL 👏

**Pull request merged**